<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgPenaEntrada</class>
 <widget class="QDialog" name="dlgPenaEntrada">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Penados</string>
       </property>
       <property name="buddy">
        <cstring>txtPenados</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="txtDuracion"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tipo</string>
       </property>
       <property name="buddy">
        <cstring>txtTipos</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Duraci&amp;ón</string>
       </property>
       <property name="buddy">
        <cstring>txtDuracion</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="txtTipos"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="txtPenados"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Penas</string>
           </property>
           <property name="buddy">
            <cstring>lwTipos</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btEliminarPenas">
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Penados</string>
           </property>
           <property name="buddy">
            <cstring>lwPenados</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btEliminarPenados">
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QListWidget" name="lwTipos"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QListWidget" name="lwPenados"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Infos extras</string>
      </attribute>
      <widget class="WidgetExtraInfos" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>321</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Texto pena</string>
       </property>
       <property name="buddy">
        <cstring>txtPenaTexto</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtPenaTexto">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Abso&amp;lución</string>
       </property>
       <property name="buddy">
        <cstring>txtAbsolucion</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtAbsolucion"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Seguridad</string>
       </property>
       <property name="buddy">
        <cstring>spSeguridad</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spSeguridad"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&amp;Motivo</string>
       </property>
       <property name="buddy">
        <cstring>txtMotivo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtMotivo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetExtraInfos</class>
   <extends>QWidget</extends>
   <header>widgets/widgetextrainfos.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtTipos</tabstop>
  <tabstop>txtDuracion</tabstop>
  <tabstop>txtPenados</tabstop>
  <tabstop>txtPenaTexto</tabstop>
  <tabstop>txtAbsolucion</tabstop>
  <tabstop>spSeguridad</tabstop>
  <tabstop>txtMotivo</tabstop>
  <tabstop>btOK</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btEliminarPenas</tabstop>
  <tabstop>btEliminarPenados</tabstop>
  <tabstop>lwTipos</tabstop>
  <tabstop>lwPenados</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
